static/app/components/u2f/u2finterface.tsx

```diff
@@ -315,7 +340,13 @@ class U2fInterface extends React.Component<Props, State> {
         className={
           'u2f-box' +
           (this.state.hasBeenTapped ? ' tapped' : '') +
-          (this.state.deviceFailure ? ' device-failure' : '')
+          (this.state.deviceFailure
+            ? this.state.failCount === 1 &&
+              navigator.userAgent.indexOf('Safari') !== -1 &&
+              navigator.userAgent.indexOf('Chrome') === -1
```

Comment:
You can use `includes` on strings I think

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes